src/controllers/llmo.js

```diff
@@ -196,13 +205,98 @@ function LlmoController() {
     return ok(config.getLlmoConfig().questions);
   };
 
+  // Handles requests to the LLMO customer intent endpoint, returns customer intent array
+  const getLlmoCustomerIntent = async (context) => {
+    const { llmoConfig } = await getSiteAndValidateLlmo(context);
+    return ok(llmoConfig.customerIntent || []);
+  };
+
+  // Handles requests to the LLMO customer intent endpoint, adds new customer intent items
+  const addLlmoCustomerIntent = async (context) => {
+    const { log } = context;
+    const { site, config } = await getSiteAndValidateLlmo(context);
+
+    const newCustomerIntent = context.data;
+    if (!newCustomerIntent || !Array.isArray(newCustomerIntent)) {
+      return badRequest('Customer intent must be provided as an array');
+    }
+
+    // Validate structure of each customer intent item
+    for (const intent of newCustomerIntent) {
+      if (!intent.key || !intent.value) {
```

Comment:
you can use the `hasText` utility function available in `spacecat-shared-utils` instead of using `!` which might evaluate to wrong truthy conditions